uitools/examples/qml_quick/src/demos/CalloutDemoForm.qml

```diff
@@ -51,5 +52,6 @@ DemoPage {
                 }
             }
         }
+//! [Set up Callout QML]
```

Comment:
🙃 I'm seing a mix of indentaion in this PR. I don't mind which way round it goes, but please choose an indentation for the snippt lines which can be applied to all examples.